.travis.yml

```diff
@@ -17,25 +17,42 @@ env:
 cache:
   directories:
     - "$CACHE_ROOT_DIR"
+    - vendor
```

Comment:
Why we cache it, but then delete it?

Reply:
Vendor in the cache is used here to pass it between stages, to avoid multiple `glide install` calls. Deleting it everytime is not required.